scripts/abi_go_bindings.sh

```diff
@@ -57,21 +57,23 @@ if [[ -z "${CONTRACT_LIST}" ]]; then
     contract_names=($DEFAULT_CONTRACT_LIST)
 fi
 
-cd $TELEPORTER_PATH
+cd $TELEPORTER_PATH/contracts/src
 for contract_name in "${contract_names[@]}"
 do
+    path=$(find . -name $contract_name.sol)
+    dir=$(dirname $path)
     abi_file=$TELEPORTER_PATH/contracts/out/$contract_name.sol/$contract_name.abi.json
     if ! [ -f $abi_file ]; then
         echo "Error: Contract $contract_name abi file not found"
         exit 1
     fi
 
     echo "Generating Go bindings for $contract_name..."
-    mkdir -p $TELEPORTER_PATH/abis-bings
+    mkdir -p $TELEPORTER_PATH/abis-bings/$dir/$contract_name
```

Comment:
The PR said we were using `abi-bindings` but this is `abis-bings`

Reply:
fixed